vms/proposervm/post_fork_block_test.go

```diff
@@ -890,8 +890,8 @@ func TestBlockAccept_PostForkBlock_TwoProBlocksWithSameCoreBlock_OneIsAccepted(t
 	if err != nil {
 		t.Fatal("could not build proBlk2")
 	}
-	if proBlk1.ID() == proBlk2.ID() {
-		t.Fatal("proBlk1 and proBlk2 should be different for this test")
+	if proBlk1.ID() != proBlk2.ID() {
+		t.Fatal("proBlk1 and proBlk2 should not be different for this test")
```

Comment:
I don't recall if you answered about this in the previous PR, so my question still stands: Why was this test inverted? It's usually not a great sign when the only change to the test is that we now expect the opposite to happen 😄 It feels to me like this means that the original stuff we were testing is no longer true, so there was a breaking change, and we should write another test to replace it, to make sure that the new test covers the new behavior.

Reply:
I am not fully sure what is the best thing to do here. `pChainHeight` was increased and a block was created. But since it is no longer supposed to increase, it is not surprising why `proBlk1.ID() == proBlk2.ID()` holds true.  
I should ask @awfm9

Reply:
This is definitely a mistake. This is like fixing a test that suddenly breaks by expecting it to error. Even though it shouldn't.